sql/driver/conn.go

```diff
@@ -53,7 +55,18 @@ func (c *conn) Exec(stmt string, args []driver.Value) (driver.Result, error) {
 	if err != nil {
 		return nil, err
 	}
-	return driver.RowsAffected(len(result.Rows)), nil
+	switch t := result.GetUnion().(type) {
+	case nil:
+		return nil, nil
+	case *Response_Result_DDL_:
```

Comment:
Oh, I didn't realize that `ResultNoRows` returns an error. Why would you want to use that? Won't we always know how many rows were affected by a DDL statement?